src/views/Home.vue

```diff
@@ -333,10 +343,48 @@ export default class Home extends Vue {
           this.error = "There was an error in adding the document.";
           return;
         });
-      this.confirm = "";
+        let day = ""
+        switch (this.time.getDay()) {
+          case 0:
+            day = "Sunday";
+            break;
+          case 1:
+            day = "Monday"
+            break;
+          case 2:
+            day = "Tuesday"
+            break;
+          case 3:
+            day = "Wednesday"
+            break;
+          case 4:
+            day = "Thursday"
+            break;
+          case 5:
+            day = "Friday"
+            break;
+          case 6:
+            day = "Saturday"
+        }
+        const postData =
+        {
+          time: this.roundToQuarter(this.time),
+          cardio: this.cardio,
+          weights: this.weights
+        }
+        const options = {
+          headers: {'Content-Type': 'application/json'}
+        }
+        let url = 'http://localhost:5000/updateLiveAverages' + '?id=' + current_gym + '&day=' + day;
```

Comment:
You should setup [proxy](https://cli.vuejs.org/config/#devserver-proxy) instead of doing this